samples/highcharts/data-tools/datacursor-group/demo.js

```diff
@@ -0,0 +1,148 @@
+const container1 = document.querySelector('#container1');
+const container2 = document.querySelector('#container2');
+const container3 = document.querySelector('#container3');
+// Main table with cursors
+const table1 = new Highcharts.DataTable({
+    columns: {
+        Rank: [1, 2, 3, 4, 5, 6],
+        City: ['Sydney', 'Melbourne', 'Brisbane', 'Perth', 'Adelaide', 'Canberra'],
+        Population: [5259764, 4976157, 2568927, 2192229, 1402393, 453558]
+    }
+});
+// Resorted tables
+const sort2 = new Highcharts.DataModifier.types.Sort({
```

Comment:
Wouldn't it make more sense if we have 3 different tables with the same column names but different values of Population? 
Each table might represent one year. IMO this might be more natural than sorted values.
Then instead of the `Rank` We can look for the specific city?

Reply:
If we talk about sense then we should have something that is applicable to the group concept. So some other demo data is needed. Though I think the time is spent better somewhere else because this is an internal demo for the technical part.

Reply:
Sure!

Reply:
But I agree with your comment. It is not ideal.